code-analysis/crates/solidity/outputs/typescript_lib/__test__/index.spec.ts

```diff
@@ -1,12 +1,12 @@
 import test from "ava";
 
-import { Language, Rule, Token, CSTNodeType, LexNodeType, LexNamedNode, CSTRuleNode } from "../index";
+import { Language, Rule, Token, CSTNodeType, LexNodeType, CSTRuleNode, CSTTokenNode } from "../index";
 
 test("parse some token", (t) => {
   const l = new Language("0.18.3");
```

Comment:
Unrelated to this PR: `0.18.3` is not a valid version. Should our API be rejecting it in this case?

Reply:
Hmmm. Yes, probably.